### PR TITLE
implemented gestures on NFT FullScreen view in Pylons

### PR DESCRIPTION
**Pull request by faizhashmiRnS**
## Description
This PR  should implement the feature:
1. #988

### CODE REVIEW

**Comment by faisalnaveedRNS** [comment]
@kjawadDeveloper2 please review

File: wallet/lib/pages/purchase_item/purchase_item_screen.dart [hunk 1]
```diff
@@ -190,6 +209,77 @@ class _PurchaseItemContentState extends State<PurchaseItemContent> {
       }
     }
   }
+
+  void _onVerticalDragStart(DragStartDetails details) {
```
Comment (kjawadDeveloper2) [inline wallet/lib/pages/purchase_item/purchase_item_screen.dart:213]: We should have one widget that encompasses this gestures logic and that we can use on purchase and owner

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]

**Review by faddat — APPROVED** [review]

**Comment by faddat** [comment]
I am good with this when jawad's feedback is addressed

File: wallet/lib/pages/purchase_item/purchase_item_screen.dart [hunk 2]
```diff
@@ -126,7 +131,7 @@ class _PurchaseItemContentState extends State<PurchaseItemContent> {
 
     return NftImageWidget(
       url: thumbnailUrl,
-      opacity: 0.4,
+      opacity: viewModel.isViewingFullNft ? 0.0 : 0.4,
```
Comment (kjawadDeveloper2) [inline wallet/lib/pages/purchase_item/purchase_item_screen.dart:134]: What you mean by isViewingFullNFT? Can you also attach a demo video please.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]

File: wallet/lib/pages/purchase_item/purchase_item_screen.dart (hunk 2 above)
Comment (ahmadrns) [reply wallet/lib/pages/purchase_item/purchase_item_screen.dart:134]: https://user-images.githubusercontent.com/106388520/190639541-f2c43480-11f6-41ee-8fa4-b514ac60d71e.mp4

Comment (ahmadrns) [reply wallet/lib/pages/purchase_item/purchase_item_screen.dart:134]: > What you mean by isViewingFullNFT? Can you also attach a demo video please.

We have declared a variable which keeps track of long Press, which means user wants to view full NFT

**Review by kjawadDeveloper2 — APPROVED** [review]

**Pull request merged**